src/lib/S3ToSalesforceUploader.ts

```diff
@@ -1,22 +1,21 @@
-// @flow
 import type { Salesforce, Folder } from './salesforceAuthenticator';
 import type { S3Folder } from './storage';
 import { getObject } from './storage';
 import util from 'util';
 
 export type sfDestination = {
-	sfDescription: string,
-	sfFolder: Folder,
-	fileName: string,
+	sfDescription: string;
+	sfFolder: Folder;
+	fileName: string;
 };
 
 export type FileUpload = {
-	destination: sfDestination,
-	fileData: any,
+	destination: sfDestination;
+	fileData: any;
```

Comment:
```suggestion
	fileData: unknown;
```

Reply:
Thank you for the comment.

I have removed the `any` [here](https://github.com/guardian/fulfilment-lambdas/pull/215/commits/ef6a1fbedd94f4f66dadafc40f0cd62305a9be0d).